src/components/Chip/Chip.tsx

```diff
@@ -78,6 +78,7 @@ const Container = styled.span<ContainerProps>`
       @media ${device.tablet} {
         > *:first-of-type:not(style) {
           display: flex;
+          place-items: initial;
```

Comment:
This is a shorthand for `align-items` and `justify-items`. Since above we only set align-items, isn't `align-items: initial;` enough?

Reply:
Indeed! thanks